app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -30,19 +32,56 @@ class HomeMenu(
         object History : Item()
         object Bookmarks : Item()
         object Quit : Item()
+        object Sync : Item()
     }
 
     val menuBuilder by lazy { BrowserMenuBuilder(menuItems) }
 
     private val hasAccountProblem get() = context.components.backgroundServices.accountManager.accountNeedsReauth()
     private val primaryTextColor =
         ThemeManager.resolveAttribute(R.attr.primaryText, context)
+    private val syncDisconnectedColor = ThemeManager.resolveAttribute(R.attr.syncDisconnected, context)
```

Comment:
@Baron-Severin I know you had showed this to me before, but just checking these two different ways of getting the sync colors are the ways that work with the highlightable item?

Reply:
I double checked, and yeah they're correct.